.gitattributes

```diff
@@ -1,18 +1,18 @@
 # Folders
 /.ddev/ export-ignore
+/.github/ export-ignore
+/.vscode/ export-ignore
 /tests/ export-ignore
 /tools/ export-ignore
 
 # Files
-/.editorconfig export-ignore
-/.gitattributes export-ignore
-/.gitignore export-ignore
-/.gitreview export-ignore
-/.php_cs.dist export-ignore
+/.* export-ignore
+/CODE_OF_CONDUCT.md export-ignore
```

Comment:
```suggestion
```
I don't think this file should be excluded, it also regulates support requests.

Reply:
It can be excluded as you don't need this file when you pull the package via composer.

Reply:
Yes, it can but I don't like to have it excluded for the mentioned reason. You also don't exclude the license for a similar reason.

Reply:
It's fine to exclude development related files which your other changes do.